_posts/2017-04-18-DockerCon-Multi-Stage-Builds-And-More.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+authors: [tom_verelst]
+title: "DockerCon 2017: Multi-Stage Builds and More"
+image: /img/dockercon2017/thumbnail.jpg
+tags: [DockerCon,Docker,Conference]
+category: Conference
+comments: true
+---
+
+DockerCon 2017 has kicked off, and Docker has changed a great deal since last year's edition.
+Docker has constantly stated that they wish to make the Docker experience as simple as possible.
```

Comment:
I think that increases readability. Obviously a minor remark.

https://en.wikipedia.org/wiki/Docker,_Inc.